src/main/java/com/powsybl/openloadflow/sa/extensions/ContingencyLoadFlowParameters.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.sa.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.contingency.Contingency;
+import com.powsybl.loadflow.LoadFlowParameters;
+
+/**
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+public class ContingencyLoadFlowParameters extends AbstractExtension<Contingency> {
+
+    private boolean distributedSlack;
+
+    private boolean areaInterchangeControl;
+
+    private LoadFlowParameters.BalanceType balanceType;
+
+    public ContingencyLoadFlowParameters(boolean distributedSlack, boolean areaInterchangeControl, LoadFlowParameters.BalanceType balanceType) {
+        this.distributedSlack = distributedSlack;
+        this.areaInterchangeControl = areaInterchangeControl;
+        this.balanceType = balanceType;
+    }
+
+    @Override
+    public String getName() {
+        return "contingency-parameters";
```

Comment:
Adapt it to the new class name

Reply:
fixed in [74c7960](https://github.com/powsybl/powsybl-open-loadflow/pull/1139/commits/74c79605b4411a6e1ddb50aa9f2a6476625fc91a)